js/event.ts

```diff
@@ -0,0 +1,257 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import * as domTypes from "./dom_types";
```

Comment:
add .ts suffix here (and below) I think

Reply:
Yup, that is it, because extensionless has been removed.

Reply:
Yeah. It only works fine when `event.ts` is building into the deno binary, but not for single import of the file for tests (which runs on deno and has to comply with deno extension requirements)

Reply:
Oh yeah... that means that anything in user land needs to be available in either the global scope or in `deno`.  So to actually fix this it needs to be a global type, so something like this in `global.ts`:

```ts
export type EventInit = event.EventInit;
```

Then in the `event_test.ts` you would just reference the type from the global namespace, no need to import anything.

Sorry for the confusion, just wasn't thinking things through.

Reply:
@hayd @kitsonk FYI the whole extension/extensionless thing seems to be infecting `dom_types` as well. When I don't have it on, it can't find the module. When I do have it on, I get:

```
..\..\js\event.ts
Error: ../../js/event.ts:2:27 - error TS2691: An import path cannot end with a '.ts' extension. Consider importing './dom_types' instead.
2 import * as domTypes from "./dom_types.ts";
                            ~~~~~~~~~~~~~~~~
```

So there seems to be an issue no matter how you slice it, but pretty much every file I see within `js/` uses extensionless.

Reply:
@acconrad for clarity, `js/event.ts` should not have extensions on import.  It is compiled by `tsc` directly and it does not support extensioned modules.  The `js/*_test.ts` files though are run by Deno which no longer supports extensions.  So those files cannot and should not directly load any files in the `js/` path directly, they should instead either rely on the global namespace or the `import * as deno from "deno";` module.  I misspoke earlier when I said that you should consider adding the extension.